cli/src/serve.js

```diff
@@ -565,6 +576,28 @@ const runCommand = (opts, done) => {
                                       Object.assign({}, { path: name }, opts.auth[name]));
       }
     }
+  }).then(() => {
+    // Automatically open up index.html in the `dist` directory only if
+    //  in dev mode and an index.html exists in the directory.
```

Comment:
Originally, it opened the browser when in dev mode. I just forgot to take this comment out.